src/usermod.c

```diff
@@ -443,6 +443,12 @@ static char *new_pw_passwd (char *pw_pass)
 		SYSLOG ((LOG_INFO, "unlock user '%s' password", user_newname));
 		memmove(pw_pass, pw_pass + 1, strlen(pw_pass) + 1);
 	} else if (pflg) {
+		if (strchr(user_pass, ':') != NULL) {
```

Comment:
Should valid_field() be called on user_pass?

Reply:
It actually is, in `lib/pwio.c` which is used to update the password entry anyway. I guess it was an issue elsewhere and I cannot reproduce it now.  I probably just completely missed that.

Reply:
I-m dropping this patch then and changing the PR name to reflect the only change left in here, sorry for the noise.